mediapipe/calculators/ovms/openvinoinferencecalculator.cc

```diff
@@ -317,7 +340,7 @@ static ov::Tensor convertTFLiteTensor2OVTensor(const TfLiteTensor& t) {
     ov::Shape shape;
     // for some reason TfLite tensor does not have bs dim
     shape.emplace_back(1);
-    // No support for scalars and no data tensors with 0-dim
+    // TODO: Support scalars and no data tensors with 0-dim
```

Comment:
We won't add that support for TFLite. Comment was left here to not leave doubts that it intentionally doesn't work and we don't have plans to change it.